mappings/net/minecraft/world/WorldServerChild.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/lq
+CLASS none/lq net/minecraft/world/WorldServerChild
```

Comment:
Just going to move it to sortme for now, a better name may come along eventually.